### PR TITLE
New HTML layout for Water Use

### PR DESCRIPTION
**Pull request by mwernimont**
Before making a pull request
----------------------------
First . . .
- [ ] Clean the code the way Vue likes it - run 'npm run lint --fix'        
- [x] Make sure all tests run

Then check for accessibly compliance
- [ ] Run WAVE plugin 508 compliance tool

Then run Browserstack; check that application works on . . .
- [x] Chrome
- [x] Safari
- [ ] Edge
- [x] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

Finally . . .
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
Created a basic HTML wireframe for the new Water Use page.

Some CSS was using CSS connected to all three pages so adjustments had to be made on them to make sure the old layouts still worked.

Basically @lindsayplatt could you see if everything still looks like you'd expect?  I checked on three browsers and several screen size and didn't notice anything, but two eyes are always better

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

**Comment by lindsayplatt** [comment]
I've tried `npm i` and `npm i npm` just to make sure I had updated packages. I still can't get this to work because of some error about `{`. 

image

**Comment by lindsayplatt** [comment]
Still no luck, but it works for Marty. Going to roll the dice, merge this, and hope that it builds on test. If not, we will go from there.
